Armadillo/src/main/java/com/scribd/armadillo/playback/MediaSessionCallback.kt

```diff
@@ -117,8 +117,8 @@ internal class MediaSessionCallback(private val onMediaSessionEventListener: OnM
         when (playbackInfo?.playbackState) {
             PlaybackState.PLAYING -> onPause()
             PlaybackState.PAUSED -> onPlay()
-            PlaybackState.NONE -> { /* do nothing */
-            }
+            PlaybackState.NONE -> Unit
+            null -> Unit
```

Comment:
Personally I'd merge the last two cases these since they will likely always need to return the same value.